homeassistant/components/ozw/light.py

```diff
@@ -198,7 +216,11 @@ async def async_turn_on(self, **kwargs):
                 0,
                 min(
                     255,
-                    round((TEMP_COLOR_MAX - round(color_temp)) / TEMP_COLOR_DIFF * 255),
+                    round(
+                        (self.max_mireds - color_temp)
```

Comment:
Let's use the internal attributes consistently now that we update them.

Reply:
Sounds good. They've been put back in.